packages/paypal-commerce-integration/src/paypal-commerce-types.ts

```diff
@@ -337,6 +337,21 @@ export interface ApproveCallbackActions {
     order: {
         get: () => Promise<PayPalOrderDetails>;
     };
+    restart(): void;
+}
+
+export interface ProviderError extends Error {
+    errors?: ErrorElement[];
+    status?: string;
+}
+
+interface ErrorElement {
+    code: string;
+    message: string;
+    // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Could you please add this rule to `.eslintrc.json`. Looks like we need this rule for the whole package for the future

Reply:
Removed rule as it is unnecessary